akka/shopping-cart-java/kubernetes/shopping-cart-service-cr.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: "v1"
+kind: "Namespace"
+metadata:
+  name: "shopping"
+---
+apiVersion: akka.lightbend.com/v1
+kind: AkkaMicroservice
+metadata:
+  name: shopping-cart-service
+  namespace: "shopping"
+spec:
+  replicas: 1
+  image: <docker-registry>/shopping-cart-service:<tag> # <1>
+  javaOptions: "-Xlog:gc -XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75"
+  resources:
+    limits:
+      memory: "2Gi"
+    requests:
+      memory: "2Gi"
+      cpu: "1"
+  jdbc:
+    credentialsSecret: shopping-cart-service-jdbc-secret
+  kafka:
+    credentialsSecret: shopping-cart-service-kafka-secret  # <2>
```

Comment:
This YAML should include the `prometheusPort: 9001` (https://github.com/lightbend/akka-platform-operator/pull/505) since this sample is explicitly for Cinnamon.

Reply:
Done.